apps/src/templates/multiple-sections-assigner.module.scss

```diff
@@ -23,17 +23,10 @@
   }
 
   .sectionListOptionsContainer {
-    &:focus {
```

Comment:
All changes in this file are just "undoing" an initial fix that forced just the sections to be scrollable.  However this fix still led to issues for teachers who were super scrolled in thus this new fix was invented.  I experimented with having two scrollers (one for sections, one for the whole dialog) but it was kinda hard to get to the "whole dialog" scroll when on the "sections scroll"